mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/api/akka/route/MasterApiMetrics.java

```diff
@@ -24,13 +24,15 @@ public class MasterApiMetrics {
     private final Counter resp2xx;
     private final Counter resp4xx;
     private final Counter resp5xx;
+    private final Counter incomingRequestCount;
```

Comment:
It looks like this is across all requests. Is there a way to emit this with the route path as a tag?

Reply:
This is used at the initial connection point so the path could be anything (cardinality not controlled by us). we have some route-level metrics already at each route level but it's up to the route to decide whether to log so I want something in general for all connections (plus track if something keeps requesting for invalid paths).